IO/BeagleBone/CANBusBBB.cs

```diff
@@ -152,15 +236,29 @@ internal CANBusBBB(string CanName)
 		/// <param name="Data">Payload of CAN Frame. Must be at most 8 bytes.</param>
```

Comment:
Typically in our documentation we have `<tag> Description </>` (note the spaces). Also, if we can reasonably make the `<summary>` tag a one-liner, we try to do so. This happens a couple times in this PR. If you could fix this in all the relevant files, that would be great :)